src/components/AmountFilter.jsx

```diff
@@ -7,12 +7,15 @@ import AmountTextField from './AmountTextField';
 
 const useStyles = makeStyles((theme) => ({
   buttonStyle: {
-    margin: theme.spacing(1),
-    height: 57,
+    margin: `${theme.spacing(2)}px ${theme.spacing(1)}px 0`,
```

Comment:
This looks pretty hacky to get this to line up with the other dropdowns. This is also very specific to the FilterBar case and may not be applicable on mobile when a filters dialog or something is shown. On desktop if you select a value from the other dropdowns the alignment looks off again. I'd leave it at a margin of `theme.spacing(1)` for now to be less hacky and align a little better. Eventually once the other dropdowns get replaced this'll look better aligned with them if they also have the same margin.